src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The link to the stylesheet should be a CSS file, not SCSS. The browser does not understand SCSS syntax. You need to compile your SCSS to CSS and then link the CSS file here.